manifests/init.pp

```diff
@@ -64,6 +67,7 @@
   $use_embedded_ruby          = false,
   $api_client_config          = {},
   $use_num_occurrences_filter = false,
+  $environment                = '',
```

Comment:
Please just remove $region on :63 and use $environment in its place

I don't want to have $region, $datacenter, and $environment.